source/includes/steps-ui-export.yaml

```
@@ -0,0 +1,27 @@
+title: Navigate to Your Realm App
+ref: navigate-to-realm-app
+content: |
+  Go to the Project Dashboard where you want to export the {+app+}. Select 
+  the :guilabel:`Realm` tab. Select the tile for the {+app+} you watn to export.
```

```suggestion
  the :guilabel:`Realm` tab. Select the title for the {+app+} you want to export.
```
necessary fix: 
tile -> title?
watn -> want

---

Spimd gppd